emojivoto-web/web/web.go

```diff
@@ -78,6 +82,16 @@ func (app *WebApp) leaderboardHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+type telepresenceIdHeader struct{}
+
+func TelepresenceInterceptIdInterceptor(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
+	value, ok := ctx.Value(telepresenceIdHeader{}).(string)
+	if ok {
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-telepresence-intercept-id", value)
```

Comment:
We should extract the header to a constant since we're repeating it in plain text